uaclient/event_logger.py

```diff
@@ -9,11 +9,12 @@
 import enum
 import json
 import sys
-from typing import Dict, List, Optional, Set  # noqa: F401
+from typing import Dict, List, Optional, Set, Union  # noqa: F401
 
 from uaclient.status import format_machine_readable_output
 
 JSON_SCHEMA_VERSION = "0.1"
+EventErrorType = Union[Optional[str], Dict[str, str]]
```

Comment:
This is definitely better than `Any` :)

A small suggestion that is IMO slightly more readable but essentially the same:
```suggestion
EventFieldValueType = Optional[Union[str, Dict[str, str]]]
```

Reply:
Just changing the name to `EventFieldErrorType´, since I think the proposed name may imply that all event fields follow that rule. But let me know if you disagree